ibm_catalog.json

```diff
@@ -134,42 +134,76 @@
               "on_by_default": true
             },
             {
-              "name": "deploy-arch-ibm-observability",
-              "description": "Enable to provision and configure IBM Cloud Logs, Cloud Monitoring, Metrics routing and Activity Tracker event routing for analysing logs and metrics generated by the SCC Workload Protection instance.",
+              "name": "deploy-arch-ibm-cloud-logs",
+              "description": "Provision and configure IBM Cloud Logs (ICL) instance which can be used for analysing logs generated by SCC Workload Protection instance. ",
```

Comment:
Can you be consistent with what was used in https://github.com/terraform-ibm-modules/terraform-ibm-app-configuration/pull/269 ? We want to try to keep this description short and clear